.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2023,
+    "sourceType": "module"
+  }
+}
```

Comment:
For some reason, we never had an eslint config file at the top level of the repo. This is needed for scripts/update-angular-dependency-version.mjs and prettier.config.mjs.